src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -365,8 +365,9 @@ private ImmutableArray<ParameterSymbol> MakeParameters(
                 else if (p < numDelegateParameters)
                 {
                     type = parameterTypes[p];
-                    refKind = RefKind.None;
-                    scope = ScopedKind.None;
+                    refKind = unboundLambda.RefKind(p);
+                    scope = unboundLambda.DeclaredScope(p);
+                    paramSyntax = unboundLambda.ParameterSyntax(p);
```

Comment:
Shouldn't we assign these values regardless of whether `p < numDelegateParameters`?

Reply:
well, if `p is >= numDelegateParameters`, what would we set them to, except some default value (which is what is going on).

Reply:
It looks like `p` is the index into the parameters provided in the lambda, and `numDelegateParameters` is the number of parameters from the delegate type. Could the lambda have more parameters than the delegate type?

Reply:
ah.  sorry.  i see what you mean.  let me try that out...